data/mods/My_Sweet_Cataclysm/sweet_monster_groups.json

```diff
@@ -33,5 +33,15 @@
     "name": "GROUP_EGG_CAFFGUM",
     "type": "monstergroup",
     "default": "mon_gum_spider_wolf"
+  },
+  {
+    "name": "GROUP_COWS",
```

Comment:
My sweet cataclysm monsters spawn from sweets and junk food that's the main gimmick of the mod. You need to make a `GROUP_EGG_CHOC_COW` group that spawn chocolate calf and find/add the item that will use it. Maybe chocolate milk could spawn chocolate calves.